app/game.rb

```diff
@@ -0,0 +1,33 @@
+require_relative './item'
```

Comment:
[OPTIONAL] I would recommend defining a name convention for your files, for example some files have `_methods.rb` and some don't, some are in plural and others in singular, some have `_list.rb` and some don't, this would make your project easier to maintain and understand by other developers.